Syntaxes/Julia.tmLanguage

```diff
@@ -244,13 +244,13 @@
 			<array>
 				<dict>
 					<key>match</key>
-					<string>\b(?:function|stagedfunction|type|immutable|macro|quote|abstract|bitstype|typealias|module|baremodule|new)\b</string>
```

Comment:
`in` is not a keyword. It's a normal function. The only thing special about it (and now `isa`) is that it can be used as an binary operator. OTOH, `new` is a keyword. It's function is very distinct from any normal function and cannot be implemented as a function.

The code here used to highlight the majority of the use of `new` correctly and removing it will cause it to highlight more code incorrectly than the confusion it fixed. So while adding it as a new highlighted keyword is questionable, it shouldn't be removed from the current list.

I don't think it's easy to make it context dependent but if anyone is bothered enough, the easiest solution is to highlight only `new(` and `new{`. These are also the set that will guarantee to break if moved in/out of inner constructor context.